index.html

```diff
@@ -1,269 +1,360 @@
+
+    <ol class="num-list">
+      <li class="num-list__item">Num list</li>
+      <li class="num-list__item">Num list</li>
+      <li class="num-list__item">Num list</li>
+    </ol>
+
+    <ul>
+      <li>
+        <a href="#" class="btn btn--primary"><span>Primary</span></a>
+        <a href="#" class="btn btn--primary btn--sm"><span>Primary</span></a>
+        <a href="#" class="btn btn--primary btn--lg"><span>Primary</span></a>
+        <a href="#" class="btn btn--primary btn--lg btn--disabled"><span>Primary</span></a>
+      </li>
+      <li>
+        <a href="#" class="btn btn--secondary"><span>Primary</span></a>
+        <a href="#" class="btn btn--secondary btn--sm"><span>Primary</span></a>
+        <a href="#" class="btn btn--secondary btn--lg"><span>Primary</span></a>
+        <a href="#" class="btn btn--secondary btn--lg btn--disabled"><span>Primary</span></a>
+      </li>
+      <li>
+        <a href="#" class="btn btn--tertiary"><span>Primary</span></a>
+        <a href="#" class="btn btn--tertiary btn--sm"><span>Primary</span></a>
+        <a href="#" class="btn btn--tertiary btn--lg"><span>Primary</span></a>
+        <a href="#" class="btn btn--on-dark btn--lg btn--disabled"><span>Primary</span></a>
+      </li>
+    </ul>
+
+    <form class="form" action="#">
+      <div class="form__row">
+        <div class="text-input">
+          <input class="text-input__focus" type="text" placeholder="Default">
+        </div>
+        <div class="text-input">
+          <input class="text-input__focus" type="search">
+        </div>
+        <div class="text-input text-input--disabled">
+          <input placeholder="Disabled" class="text-input__focus text-input__focus--disabled" type="search">
+        </div>
+        <div class="text-input" data-components="Component1">
+          <input class="js-datepicker text-input__focus">
+        </div>
+      </div>
+      <div class="form__row">
+        <div class="form__radio">
+          <div class="form__radio-holder">
+            <label class="radio-input">
+              <input class="radio-input__focus"  type="radio" name="radio01" checked="checked">
+              <span class="radio-input__label">Radio</span>
+            </label>
+          </div>
+          <div class="form__radio-holder">
+            <label class="radio-input">
+              <input class="radio-input__focus"  type="radio" name="radio01" checked="checked">
+              <span class="radio-input__label">Radio</span>
+            </label>
+          </div>
+        </div>
+      </div>
+      <div class="form__row">
+        <div class="form__check-holder">
+          <label class="check-input">
+            <input class="check-input__focus"  type="checkbox" checked="checked">
+            <span class="check-input__label">Radio</span>
+          </label>
+        </div>
+        <div class="form__check-holder">
+          <label class="check-input">
+            <input class="check-input__focus"  type="checkbox">

```

Comment:
please remove two spaces between [__focus"   type="checkbox"] and leave only one